doc/getting-started/understanding-config-files.md

```diff
@@ -6,21 +6,55 @@ Tells your node to which nodes in the network it should talk to. A minimal versi
 
 ```json
 {
-  "Producers": [
+  "LocalProducers": {
+    "groups": [
+      {
+        "localRoots": {
+          "addrs": [
+            {
+              "isDomain": true,
```

Comment:
Can you get rid of 'isDomain'?  the previous config file didn't require it I don't see why the new format makes it necessary.